benchmarks/distributed/lambda_runtimes/cpp_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Create CMake project
+cmake_minimum_required(VERSION 3.9)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+#project(test-python LANGUAGES CXX)
+set(LAMBDA_NAME test-python)
+
+# Create AWS cpp Project
+find_package(aws-lambda-runtime)
+#find_package(AWSSDK COMPONENTS s3)
+add_executable(${LAMBDA_NAME} "main.cpp" "lambda_main.cc" "sighandler.cc")
+target_link_libraries(${LAMBDA_NAME} PRIVATE AWS::aws-lambda-runtime ${AWSSDK_LINK_LIBRARIES})
+
+# Link Python libraries + interpreter
+set(PYTHON_RESOURCES_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../python38_resources)
+#target_include_directories(${LAMBDA_NAME} PRIVATE ${PYTHON_RESOURCES_DIR}/include/python3.7m/)
+target_include_directories(${LAMBDA_NAME} PRIVATE /usr/include/python3.8/)
+
+#find_library(LAMBDA_PYTHON_PACKAGE NAMES python3.8) #PATHS ${PYTHON_RESOURCES_DIR}/libpy38 REQUIRED)
```

Comment:
remove all the commented lines, or add comment on what effect uncommenting would have

Reply:
removed most, commented remaining